packages/adapter-static/package.json

```diff
@@ -18,7 +18,9 @@
 		"@sveltejs/kit": "workspace:*",
 		"playwright-chromium": "^1.10.0",
 		"port-authority": "^1.1.2",
+		"svelte": "^3.40.0",
 		"sirv": "^1.0.12",
-		"typescript": "^4.3.5"
+		"typescript": "^4.3.5",
```

Comment:
I notice we have typescript declared in project root `package.json`. Is there a way to use that version here? Otherwise I'm not sure the point of declaring it in the main `package.json`

Reply:
i think that line could be removed as the only reference to typescript in this package is in the `"check": "tsc"`  script. 

didn't search for extraneous declarations for this PR, just added the ones required to keep everything (build, test, format, lint) working.

Reply:
It looks like every package declares a dependency on TypeScript, but I don't think any of them use it other than `tsc`

Reply:
I'd say go ahead and remove it assuming `pnpm check` still runs and passes, but we should remove it from all of them

Reply:
got ahead of myself and pushed the typescript removal directly on master, added here now too.